janus.h

```diff
@@ -105,6 +107,10 @@ void janus_session_handles_clear(janus_session *session);
  * @param[in] session The Janus Core-Client session
  * @returns The JSON array */
 json_t *janus_session_handles_list_json(janus_session *session);
+/*! \brief Method to set session timeout value.
+ * @param[in] session The Janus Core-Client session
+ * @param[in] timeout Timeout value in seconds. */
+void janus_session_set_timeout(janus_session *session, uint timeout);
```

Comment:
We don't really need the function here, I think. It's probably my fault as initially I put a lot of functions in this header that really didn't need exposing, as they're only used within `janus.c` anyway.